libfwupd/fwupd-enums.c

```diff
@@ -267,6 +271,10 @@ fwupd_device_flag_from_string (const gchar *device_flag)
 		return FWUPD_DEVICE_FLAG_VERSION_CHECK_REQUIRED;
 	if (g_strcmp0 (device_flag, "install-all-releases") == 0)
 		return FWUPD_DEVICE_FLAG_INSTALL_ALL_RELEASES;
+	if (g_strcmp0 (device_flag, "md-set-name") == 0)
+		return FWUPD_DEVICE_FLAG_MD_SET_NAME;
```

Comment:
What do you envision that this would be used for instead of the category one?

Reply:
Two things came to mind: DFU devices and also USB hubs that are actually docking stations.